### DISCUSSION TITLE
Suggestion to remove the unofficial successor label in description.

### DISCUSSION THREAD

**Discussion by shrihari-prakash** [node-oauth/node-oauth2-server#270] [General]
![image](https://github.com/node-oauth/node-oauth2-server/assets/35889246/4424b98e-a817-4b17-8352-e1ae727a0c6c)
I think mentioning the library as Unofficial Successor in the about section would make people question the library's reputation and active maintenance. It might be good to mention this in README, but perhaps a good idea to remove from the about section which is the very first section people read.

**Comment by jankapunkt** [comment]
I think we added this to avoid using a 1:1 about section that actually sounds the same as the oauthjs one

**Reply by shrihari-prakash** [reply]
Hmm, I understand the need for it to be different from the oauthjs one. Perhaps good idea to rephrase the description maybe? In any case, you have also put tremendous efforts into refactoring and modernizing the library. It is far from the original one now 🙂.

**Reply by shrihari-prakash** [reply]
It could be a turn off for some people to see the word unofficial. Might have the risk of making the library look not maintained.

**Reply by jankapunkt** [reply]
I removed "unofficial" for now but I think we may find a more crisp and positive description.

**Reply by shrihari-prakash** [reply]
"Robust, battle-tested and compliant Node.js OAuth2 server with native async/await and PKCE support."?

**Comment by shrihari-prakash** [comment]
Closing as description has been updated.
